src/platform/linux/misc.cpp

```diff
@@ -518,34 +520,44 @@ std::unique_ptr<deinit_t> init() {
     window_system = window_system_e::X11;
   }
 #endif
+
 #ifdef SUNSHINE_BUILD_CUDA
-  if(verify_nvfbc()) {
-    sources[source::NVFBC] = true;
+  if(config::video.capture.empty() || config::video.capture == "nvfbc") {
+    if(verify_nvfbc()) {
+      sources[source::NVFBC] = true;
+    }
   }
 #endif
 #ifdef SUNSHINE_BUILD_WAYLAND
-  if(verify_wl()) {
-    sources[source::WAYLAND] = true;
+  if(config::video.capture.empty() || config::video.capture == "wlr") {
+    if(verify_wl()) {
+      sources[source::WAYLAND] = true;
+    }
   }
 #endif
 #ifdef SUNSHINE_BUILD_DRM
-  if(verify_kms()) {
-    if(window_system == window_system_e::WAYLAND) {
-      // On Wayland, using KMS, the cursor is unreliable.
-      // Hide it by default
-      display_cursor = false;
+  if(config::video.capture.empty() || config::video.capture == "kms") {
+    if(verify_kms()) {
+      if(window_system == window_system_e::WAYLAND) {
+        // On Wayland, using KMS, the cursor is unreliable.
+        // Hide it by default
+        display_cursor = false;
+      }
     }
 
     sources[source::KMS] = true;
   }
 #endif
 #ifdef SUNSHINE_BUILD_X11
-  if(verify_x11()) {
-    sources[source::X11] = true;
+  if(config::video.capture.empty() || config::video.capture == "x11") {
+    if(verify_x11()) {
+      sources[source::X11] = true;
+    }
   }
 #endif
 
   if(sources.none()) {
+    BOOST_LOG(error) << "Unable to initialize capture method";
```

Comment:
Should this be `fatal` instead of `error`?

Reply:
I have something in mind for a follow up PR. Currently if you configure a capture method that does not work, Sunshine will shut down and you will lose the web UI. I would like it to avoid exiting so that you are still able to edit configuration even if the capture is not working.